openy.install

```diff
@@ -168,3 +168,82 @@ function openy_update_8011() {
+  // Enable module.
+  \Drupal::service('module_installer')->install([
+    'metatag',
+    'redirect',
+    'simple_sitemap'
+  ]);
+
+  // Revert individual configurations
+  // Image Style & OG defaults
+  $config_dir = drupal_get_path('module', 'openy_node') . '/config/install';
+  $config_importer = \Drupal::service('openy_upgrade_tool.importer');
+  $config_importer->setDirectory($config_dir);
+  $config_importer->importConfigs([
+    'image.style.facebook',
+    'metatag.metatag_defaults.node',
+    'field.storage.node.field_meta_tags',
+  ]);
+  $config = drupal_get_path('module', 'openy_node') . 'openy_node.info.yml';
+  $config_importer = \Drupal::service('openy_upgrade_tool.param_updater');
+  $config_importer->update($config, 'openy_node.info', 'dependencies');
+
+  // Node metatags and sitemap config
+  $features = [
+    'openy_loc_branch' => 'branch',
+    'openy_loc_camp' => 'camp',
+    'openy_loc_facility' => 'facility',
+    'openy_node_blog' => 'blog',
+    'openy_node_landing' => 'landing_page',
+    'openy_node_program' => 'program',
+  ];
+  foreach ($features as $module => $bundle) {
+    // Metatag
+    // Field definitions
+    $config_dir = drupal_get_path('module', $module) . '/config/install';
+    $config_importer = \Drupal::service('openy_upgrade_tool.importer');
+    $config_importer->setDirectory($config_dir);
+    $config_importer->importConfigs([
+      'field.field.node.' . $bundle . '.field_meta_tags',
+      'simple_sitemap.bundle_settings.' . $bundle . '.landing_page',
+    ]);
+    // Dependencies
+    $config = drupal_get_path('module', $module) . $module . '.info.yml';
+    $config_importer = \Drupal::service('openy_upgrade_tool.param_updater');
+    $config_importer->update($config, $module . '.info', 'dependencies');
+    // Entity form
+    $config = drupal_get_path('module', $module) . '/config/install/core.entity_form_display.node.' . $bundle . '.default.yml';
+    $config_importer = \Drupal::service('openy_upgrade_tool.param_updater');
+    $config_importer->update($config, 'core.entity_form_display.node.' . $bundle . '.default', 'dependencies.config');
+    $config_importer->update($config, 'core.entity_form_display.node.' . $bundle . '.default', 'dependencies.module');
+    $config_importer->update($config, 'core.entity_form_display.node.' . $bundle . '.default', 'content.field_meta_tags');
+    // Entity view display
+    $config = drupal_get_path('module', $module) . '/config/install/core.entity_view_display.node.' . $bundle . '.default.yml';
+    $config_importer = \Drupal::service('openy_upgrade_tool.param_updater');
+    $config_importer->update($config, 'core.entity_view_display.node.' . $bundle . '.default', 'dependencies.config');
+    $config_importer->update($config, 'core.entity_view_display.node.' . $bundle . '.default', 'dependencies.module');
+    $config_importer->update($config, 'core.entity_view_display.node.' . $bundle . '.default', 'content.field_meta_tags');
```

Comment:
All these updates should live in appropriate modules.

E.g. update for "openy_loc_branch" should be in `openy_loc_branch` module and etc.